student.rb

```diff
@@ -1,9 +1,11 @@
 require_relative 'person'
 
 class Student < Person
+  attr_accessor :classroom
+
   def initialize(age, classroom, name: 'Unknown', parent_permission: true)
     super(age, name: name, parent_permission: parent_permission)
-    @classroom = classroom
+    classroom.add_student(self)
```

Comment:
- [ ] The Student class should have its own distinct method for the other side of the association, just like the Classroom class.

  ```
  def classroom=(classroom)
    @classroom = classroom
    classroom.students.push(self) unless classroom.students.include?(self)
  end
  ```